src/Windowing/Silk.NET.Windowing.Common/Structs/VideoMode.cs

```diff
@@ -28,9 +28,35 @@ public VideoMode(int refreshRate)
         /// </summary>
         public int? RefreshRate { get; }
 
+        /// <summary>
+        /// Aspect ratio of the full screen window. This value will not be 100% accurate to the actual value, however
+        /// is more accurate to what a user may expect the ratio to be (for example a resolution of 1366x768 will appear
+        /// as 16:9, even though it is not quite 16:9 in reality).
+        /// </summary>
+        public Vector2D<int>? AspectRatio { get; }
```

Comment:
Re discord discussions we might want to clarify that this is an Estimate and is not necessarily the true value (but gives effectively the same result). This is the precedent we want to set with mathsy things going forward (#690). If you did want to add a non-estimate this can be done in a future PR

Reply:
i.e.
```suggestion
        public Vector2D<int>? AspectRatioEstimate { get; }
```